### ISSUE TITLE
datadog dashboard: object contains unparsed element map definition background_color...

### ISSUE THREAD

**Issue by thomas-riccardi** [GoogleCloudPlatform/terraformer#1451]
`terraformer-datadog-git-e9a73ea import datadog --resources=dashboard`
```
2022/09/08 15:34:58 object contains unparsed element: map[definition:map[background_color:gray content:Read/ Write Operations font_size:18 show_tick:true text_align:center tick_edge:bottom tick_pos:50% type:note] id:7 layout:map[height:9 width:44 x:159 y:0.08333333333333333]]
```

(I tried with both v0.8.21 and go build git HEAD e9a73ea to include #1433, related to #1375: same issue)

(it's hard to find which dashboard is concerned: dashboard ids logged around the error don't seem to be related; even with `-v` I could not find more details)

(I'm not sure what are the consequences of this error log: will I lose this setting on terraform apply?)

**Comment by skarimo** [comment]
Hi, in this case, you can edit the widget and save it so the values get updated properly to `int`. The layout x and y should not be floats this is probably leftover from the past behaviors. Once the values are updated to integers this shouldn't occur. Thanks.

**Comment by thomas-riccardi** [comment]
I'm closing the issue: it's a niche issue with now documented workarounds: good enough for me.

**Comment by kschwarz1116** [comment]
I am running into this issue as well, however, the workaround does not appear to work for me. The widget in question is a Logs Stream widget. @skarimo I'm curious how you edited the widget -- I've tried both editing attributes on the widget, as well as moving the widget and moving it back to no avail. I've attempted this both in Firefox and in Chrome.

Any help appreciated -- if it makes sense, I'd like to reopen the issue as it's preventing me from using terraformer.
